src/mux.js

```diff
@@ -37,12 +37,14 @@ class Mux extends Base {
   /**
    * Mux Constructor
    *
-   * @param {string} accessToken - Mux API Access Token
+   * @param {string|options} accessToken - Mux API Access Token, or a configuration object.
    * @param {string} secret - Mux API secret
+   * @param {object} options - Optional configuration object
+   * @param {object='https://api.mux.com'} options.baseUrl - Change the base URL for API requests.
```

Comment:
shouldn't this be a string? Or is this how you swagger a string property under an object?